database_sanitizer/tests/test_dump_postgres.py

```diff
@@ -65,6 +71,30 @@ def test_sanitize():
     assert "2\t2018-01-02 00:00:00\tSanitized" in dump_output_lines
 
 
+def test_skip_table_rows():
+    url = urlparse.urlparse("postgres://localhost/test")
+    config = Configuration()
+    config.skip_rows_for_tables.append('test')
+
+    with mock.patch("subprocess.Popen",
+                    side_effect=create_mock_popen(MOCK_PG_DUMP_OUTPUT)):
+        output = list(sanitize(url, config))
+
+    assert '--- Fake PostgreSQL database dump' in output
+    assert '--- Final line after `COPY` statement' in output
+    assert 'CREATE TABLE "public"."test" (' in output
+    assert '"id" integer NOT NULL,' in output
+    assert '"created_at" timestamp with time zone NOT NULL,' in output
+    assert '"notes" character varying(255) NOT NULL' in output
+
+    assert ('COPY "public"."test" ("id", "created_at", "notes") FROM stdin;'
+            not in output)
+    assert '1\t2018-01-01 00:00:00\tTest data 1' not in output
+    assert '2\t2018-01-02 00:00:00\tTest data 2' not in output
+    assert '3\t2018-01-03 00:00:00\tTest data 3' not in output
+    assert '\\.' not in output
```

Comment:
Same here. IMHO this would be better to just assert the whole contents of the output with something like:
```python
assert output == [
    '--- Fake PostgreSQL database dump',
    '',
    'COMMENT ON SCHEMA "public" IS \'standard public schema\';',
    ...
]
```